React/React-Core.podspec

```diff
@@ -41,14 +41,27 @@ Pod::Spec.new do |s|
                              "Views/RCTRefreshControl*",
                              "Views/RCTSlider*",
                              "Views/RCTSwitch*",
+  s.private_header_files   = "Cxx*/*.h"
   s.compiler_flags         = folly_compiler_flags + ' ' + boost_compiler_flags
   s.header_dir             = "React"
   s.framework              = "JavaScriptCore"
   s.library                = "stdc++"
   s.pod_target_xcconfig    = { "HEADER_SEARCH_PATHS" => "\"$(PODS_TARGET_SRCROOT)/ReactCommon\" \"$(PODS_ROOT)/boost-for-react-native\" \"$(PODS_ROOT)/DoubleConversion\" \"$(PODS_ROOT)/Folly\"" }
+  s.default_subspec         = "Default"
+
+  s.subspec "Default" do |ss|
+    # no-op
+  end
+
+  s.subspec "CxxBridge" do |ss|
+    # Make the C++ headers visible if they are needed
+    ss.public_header_files   = "**/*.{h}"
```

Comment:
According to https://guides.cocoapods.org/syntax/podspec.html#private_header_files, it looks like `private_header_files` is computed vs `public_header_files`, so doing it this way ended up still excluding the private_header_files?

> These patterns are matched against the public headers (or all the headers if no public headers have been specified) to exclude those headers which should not be exposed to the user project and which should not be used to generate the documentation. When the library is built, these headers will appear in the build directory.

Reply:
Nvm, this seems to work. I edited the wrong file when verifying.